extension/wallet/Cargo.toml

```diff
@@ -13,13 +13,13 @@ default = ["console_error_panic_hook"]
 [dependencies]
 aes-gcm-siv = { version = "0.9", features = ["std"] }
 anyhow = "1"
-baru = { git = "https://github.com/da-kami/baru", branch = "lender-decides-timelock" }
 bip32 = { version = "0.2", features = ["secp256k1-ffi", "bip39"], default-features = false }
-coin_selection = { path = "../../coin_selection" }
+async-trait = "0.1"
+baru = { path = "/Users/rishflab/comit-network/baru"}
+bdk = { version = "0.4", default-features = false }
```

Comment:
This is a really old version of bdk. The current version is 0.8: https://github.com/bitcoindevkit/bdk/releases

Reply:
Pretty sure we don't need this `bdk` dependency here. But we can look to bump this version in `baru`'s `coin_selection` module.